tasks/epi_stats/fetch_citystats.py

```diff
@@ -29,15 +29,18 @@ def critical_districts(data):
     c1b = data_latest['total.confirmed.14_day_ratio'] > 1
     
     
-    n = 20  #################### TO n, ASSIGN NUMBER OF CITIES SATISFYING CRITERIA 1 TO BE PICKED #########
+    n = 30  #################### TO n, ASSIGN NUMBER OF CITIES SATISFYING CRITERIA 1 TO BE PICKED #########
     criteria1 = (c1a & c1b)
     critical_cities_c1 = data_latest[criteria1][['district', 'delta.confirmed']]
     critical_cities_c1.sort_values(by=['delta.confirmed'], ascending=False, inplace=True)
     critical_cities_c1_capped = critical_cities_c1.head(n)    
     
     # criteria 2
-    top20_cutoff = data_latest["total.confirmed"].sort_values(ascending=False).iloc[20]
-    c2 = data_latest['total.confirmed'] > top20_cutoff
+    n_c2 = 20    ###### ENTER number of cities as per criteria 2 HERE
+    topn_cutoff = data_latest.loc[~data_latest["district"].isin(critical_cities_c1_capped.district.drop_duplicates().to_list())]\
+                       ["delta.active"].sort_values(ascending=False).iloc[n_c2]
+
+    c2 = data_latest['delta.active'] > topn_cutoff
```

Comment:
c2 = c2.loc[~c2.district.isin(critical_cities_c1_capped.district.drop_duplicates().to_list()),:]

Reply:
have made an equivalent edit (since c2 is a pandas series that does not have the 'district' variable available to run the suggested line)